libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationMapRoute.java

```diff
@@ -929,7 +929,7 @@ private void drawWaypointMarkers(@NonNull MapboxMap mapboxMap, @Nullable Drawabl
                                 stop(10f, 0.8f),
                                 stop(0f, 0.6f)
                         )),
-                        PropertyFactory.iconPitchAlignment(Property.ANCHOR_MAP),
+                        PropertyFactory.iconPitchAlignment(Property.ICON_PITCH_ALIGNMENT_MAP),
```

Comment:
Is this also part of this issue?

Reply:
No, it triggers a static error:

The iconPitchAlignment requires `@Property.ICON_PITCH_ALIGNMENT`
```
public static PropertyValue<String> iconPitchAlignment(@Property.ICON_PITCH_ALIGNMENT String value) {
```

This has the following properties:
```
@StringDef({
      ICON_PITCH_ALIGNMENT_MAP,
      ICON_PITCH_ALIGNMENT_VIEWPORT,
      ICON_PITCH_ALIGNMENT_AUTO,
    })
  @Retention(RetentionPolicy.SOURCE)
  public @interface ICON_PITCH_ALIGNMENT {}
```

Reply:
I think this should be in a seperate PR. Because the pr is about opening up the initialise and the addlisteners. So could you please make them seperate?